packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -414,8 +420,13 @@ export abstract class FluidDataStoreContext
 	private readonly parentContext: IFluidParentContextPrivate;
 	public readonly storage: IDocumentStorageService;
 	public readonly scope: FluidObject;
-	// Represents the group to which the data store belongs too.
+	/**
+	 * The loading group to which the data store belongs to.
+	 */
 	public readonly loadingGroupId: string | undefined;
+	/**
+	 * ???
```

Comment:
Docs here should cover what these string mean, why it can be undefined, and why it can be reassigned later.

Reply:
I updated the comment, but I think more detail is needed as this is a string array, not a single string, and its unclear how to use this string array to look something up in a NamedFluidDataStoreRegistryEntries, or which NamedFluidDataStoreRegistryEntries this is referring to (can it link a property not a type?)

Reply:
Does this have the same semantics as the array in IFluidDataStoreContext.packagePath?